recipes/conversation_view/feed_item_row/feed_item_row_default.story.vue

```diff
@@ -14,6 +14,12 @@
       <template v-if="defaultSlot">
         <span v-html="defaultSlot" />
       </template>
+      <template #avatar>
+        <span
+          v-if="avatar"
+          v-html="avatar"
+        />
+      </template>
```

Comment:
Move the v-if to the template to maintain the same logic in all of them.
```suggestion
      <template 
        v-if="avatar"
        #avatar
      >
        <span
          v-html="avatar"
        />
      </template>
```